finality-aleph/src/party/authority.rs

```diff
@@ -65,11 +63,15 @@ impl Subtasks {
     async fn stop(self) {
         // both member and aggregator are implicitly using forwarder,
         // so we should force them to exit first to avoid any panics, i.e. `send on closed channel`
-        self.member.stop().await;
+        debug!(target: "aleph-party", "Started to stop all tasks");
+        //self.member.stop().await;
+        //trace!(target: "aleph-party", "Member stopped");
```

Comment:
Uhhh, uncomment these plx. And this also explains why we have these errors all over the logs, the ones you told me not to mention. xd

Reply:
Done